src/Wrep/Daemonizable/Command/EndlessCommand.php

```diff
@@ -212,22 +213,6 @@ private function getMemoryInfo(bool $peak = false): array
         return $info;
     }
 
-    /**
```

Comment:
@mac-cain13 I could not find any reason whatsoever to override the `setCode` function in our class. We call `parent::setCode` inside constructor. the code we execute is programmed in the derived classes inside its functions, which are called from the `runloop`. So why setCode is important? Also the function was copy of the parent function from Symfony 2, since Symfony 3 different code was used using `Closure` class. Yet another example why I think his was never used, thus the inconsistency never manifested as a bug.